### PR TITLE
Change the icon for the application

### PR DESCRIPTION
**Pull request by cardona6**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "axios": "^0.17.0",
     "bootstrap": "^4.0.0-beta",
+    "package-name": "^0.1.0",
```
Comment (kabaros) [inline client/package.json:8]: why did you add this package?

File: client/src/components/Categories/CategoryCard.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { CardLink, Row, Card, CardText, CardBody, CardTitle, Button, CardImg, Col } from 'reactstrap';
 import { Link } from 'react-router-dom';
+import CategoryIcon from './CategoryIcon';
```
Comment (kabaros) [inline client/src/components/Categories/CategoryCard.js:4]: why did you change this file `CategoryCard.js` - this shouldn't have changed.

**Review by kabaros — CHANGES_REQUESTED** [review]

File: client/package.json (hunk 1 above)
Comment (cardona6) [reply client/package.json:8]: it was by mistake

File: client/src/components/Categories/CategoryCard.js (hunk 2 above)
Comment (cardona6) [reply client/src/components/Categories/CategoryCard.js:4]: I'll fix it now and send it

Comment (cardona6) [reply client/src/components/Categories/CategoryCard.js:4]: Mozafar I just added the import at the top because when I started git pull upstream it was already in the file categoryCard.js
